.travis.yml

```diff
@@ -2,6 +2,7 @@ language: node_js
 node_js:
   - "8"
   - "10"
+  - "11"
```

Comment:
This is not really necessary. 11 is a stop-gap that will be EOL-ed in a couple of months.

In a couple of weeks, when 12 is released, we will update everything and drop support for 8.